homeassistant/components/romy/entity.py

```diff
@@ -0,0 +1,24 @@
+"""Base entity for ROMY."""
+
+from romy import RomyRobot
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN
+
+
+class RomyEntity(Entity):
```

Comment:
```suggestion
class RomyEntity(CoordinatorEntity[RomyVacuumCoordinator]):
```
If you redefine it a bit to inherit from the coordinator here, you will simplify your code here and in the following platforms. Or is there a specific reason you did it like this?

Reply:
changed, it only grow up that way cause i added coordinator first